test/tmt/main.fmf

```diff
@@ -0,0 +1,33 @@
+# Only common dependencies that are NOT required to run netavark-tests.sh are
+# specified here. Everything else is in netavark-tests.sh.
+require:
+    - cargo
+    - make
+
+adjust:
+    duration: 10m
+    when: arch == aarch64
+
+/validate:
+    tag: upstream
+    summary: Validate test
+    test: make -C ../.. validate
+
+/unit:
+    tag: upstream
+    summary: Unit test
+    test: make -C ../.. unit
+
+/upstream:
+    tag: upstream
+    summary: Run tests on upstream PRs
+    
+    /integration_test:
+        test: bash tests.sh upstream
+
+/downstream:
```

Comment:
The duplication between upstream and downstream confuses me here? Also the names make no sense as this runs integration tests not upstream/downstream specific tests. 
Can't we just assign two tags upstream and downstream to one definition?

Reply:
I pushed a test change. Let me see how I can best address this. Hold off on this for a bit.

Reply:
see [this change](https://github.com/containers/aardvark-dns/pull/465/files#diff-68ada78e2ae75611599170f255698e6e31c66e0bfe3aae153e9c41b6cb7ad89fR21-R30) which reorgs the integration tests. The results are [here](https://artifacts.dev.testing-farm.io/abce20c4-0a57-4e57-9808-82822c3e9cfa/)

Given aardvark-dns tests depend on netavark, for upstream we'll need to fetch netavark from podman-next while for downstream we'll need to fetch official fedora netavark. So, there will be some differences. We `could` move this environment detection in the script itself, but hope we're cool with doing that in a followup PR.

Reply:
yes that looks good enough to me